ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx

```diff
@@ -112,34 +105,24 @@ function RequestCVEsTable({
                             </Th>
                         </Tr>
                     </Thead>
-                    {data?.imageCVEs.length === 0 && (
-                        <Tbody>
-                            <Tr>
-                                <Td colSpan={6}>
-                                    <Bullseye>
-                                        <EmptyStateTemplate
-                                            title="No results found"
-                                            headingLevel="h2"
-                                            icon={SearchIcon}
-                                        />
-                                    </Bullseye>
-                                </Td>
-                            </Tr>
-                        </Tbody>
-                    )}
-                    {data?.imageCVEs.length !== 0 &&
-                        data?.imageCVEs.map(
-                            (
-                                {
+                    <TbodyUnified
+                        tableState={tableState}
+                        colSpan={colSpan}
+                        emptyProps={{
+                            title: 'No CVEs found',
```

Comment:
A comment in passing that this `title` has **found**
* which a reader might associate with filtering
* which differs from the pattern elsewhere in these changed files like **No pending exception requests**

Reply:
I see what you mean. Would it make more sense if it said `No CVEs in this request`? Technically, we should always have at least 1 CVE for a request so this should ideally not even show up.

Reply:
Yes, that has a nice balance between not too short and not too long. Just right!